src/Input/Input.variants.js

```diff
@@ -0,0 +1,79 @@
+import { BORDERS } from '../assets/core'
+import colors from '../assets/colors'
+import typography from '../Typography/variants'
+
+const DEFAULT_STYLE = {
+  ...typography.inputLabel,
+  height: '40px',
+  background: colors.backgroundLightGray,
+  border: `${BORDERS[1]} ${colors.inputGray}`,
+  boxSizing: 'border-box',
+  color: colors.textInputActive,
+  borderRadius: '3px',
```

Comment:
Same here, I think we already have something called `radii`

Reply:
Done